### PR TITLE
pop ups navigating back

### PR DESCRIPTION
**Pull request by AndriiHubarievW**
PR Checklist

- [ ] - Latest `master` branch merged
- [ ] - PR title descriptive (can be used in release notes)

## Description
- switched to previous implementation of back navigation for pop_ups in order to achieve needed behaviour.

_What this PR does_

## Git Issues

_Closes #_

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

### CODE REVIEW

**Review by smborio — APPROVED** [review]
reviewed in meeting. Will merge and see if the functionality works well.

**Review by chrismclarke — CHANGES_REQUESTED** [review]
I'd say best to hold off merging this for now, if I remember correctly when I tried to implement things this way there were a number of issues when it came to popups triggered from after navigation and also when triggered from within nested components.

I'm not sure if the current debug templates demonstrate this correctly or not, but I am quite sure that the code does not work exactly how we would want it to

**Pull request closed without merging**
